### PR TITLE
[issue#3] add RNG function

### LINKED ISSUE #3: Weighted lottery
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
Create a function that will generate a random number

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Benjibuns**
create new file w/ export and add RNG function fixes #9

### CODE REVIEW

File: git-play-generator/src/weightedLot.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export function weightedSlots() {
+    let chances = Math.floor(Math.random() * 5)
+    if(chances === 1) {
+        return "You win"
+    }else{
+        return "try again"
+    }
+}
+
+console.log(weightedSlots())
```
Comment (gleekzorp) [inline git-play-generator/src/weightedLot.js]: Please remove this `console.log`

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Awesome function.  It looks like it will work but let's test it inside the bootstrap instead since your exporting it we really don't want to have a `console.log` inside the `weightedLot.js` file.

File: git-play-generator/src/weightedLot.js (hunk 1 above)
Comment (Benjibuns) [reply git-play-generator/src/weightedLot.js]: Sounds good!

**Comment by Benjibuns** [comment]
I will fix those and add to boostrap

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Please change your `let` variable to `const`

**Review by gleekzorp — APPROVED** [review]
Awesome!

**Pull request merged**